config/ConfigExampleTests/AppTests.m

```diff
@@ -27,7 +27,7 @@ @implementation AppTests
 
 
 - (void)testRemoteConfig {
-  [FIRApp configure];
+  if([FIRApp defaultApp] == nil){ [FIRApp configure]; };
```

Comment:
To make this scalable, move to the setup function.

Also, fix the style to:

if ([FIRApp defaultApp] == nil) { 
  [FIRApp configure]; 
}

Reply:
Here's a Swift example of what I mean https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseRemoteConfig/Tests/SwiftAPI/APITestBase.swift#L28